packages/@glimmer/compiler/lib/builder/builder.ts

```diff
@@ -38,7 +38,7 @@ import {
   SPLAT_HEAD,
   THIS_VAR,
 } from '@glimmer/constants';
-import { assert, exhausted, expect, isPresentArray } from '@glimmer/debug-util';
+import { exhausted, expect, isPresentArray,localAssert } from '@glimmer/debug-util';
```

Comment:
Renamed `assert` to `localAssert` to make it super-clear that it's not included in the build output.